src/main/java/com/akkus/gen/VocidEntity.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2019 Abdurrahman AKKUŞ <iletisim@algoritimbilisim.com>
+ * Bu yazılımın tüm hakları Algoritim Bilişim'e aittir.
+ */
+package com.akkus.gen;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+/**
+ *
+ * @author Abdurrahman AKKUŞ 
+ */
+@Data
+@Entity
+@Table(name="vocid")
+public class VocidEntity implements ISuperEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="id")
```

Comment:
`@Column(name="id")`
is not needed when the field names are the same on the entity.